gremlin-core/src/main/java/org/apache/tinkerpop/gremlin/process/traversal/util/AndP.java

```diff
@@ -29,11 +32,19 @@
  */
 public final class AndP<V> extends ConnectiveP<V> {
 
-    public AndP(final P<V>... predicates) {
+    public AndP(final List<P<V>> predicates) {
         super(predicates);
         this.biPredicate = new AndBiPredicate(this);
     }
 
+    @Deprecated
+    /**
+     * @deprecated Since 3.2.0 -- use {@link AndP(List)}
```

Comment:
Minor nit.....We've consistently used "As of release 3.2.0-incubating, replaced by {@link SomeOtherClass#someNewMethod()}" for deprecation comments.